index.html

```diff
@@ -16,30 +16,208 @@
         </nav>
     </header>
     <main>
-        <section class="headline-section">
-            <h1>I'm Massam.<br>Glad to see you!</h1>
-            <p>I’m a software developer! I can help you build a product , feature or website 
-                Look through some of my work and experience! If you like what you see and have 
-                a project you need coded, don’t hesitate to contact me.
-            </p>
-            <label for="media-icons">LET'S CONNECT</label>
-            <ul>
-                <li>
-                    <a href="#" class="icons"><i class="fa-brands fa-twitter"></i></a>
-                </li>
-                <li>
-                    <a href="#" class="icons"><i class="fa-brands fa-linkedin"></i></a>
-                </li>
-                <li>
-                    <a href="#" class="icons"><i class="fa-sharp fa-solid fa-m"></i></a>
-                </li>
-                <li>
-                    <a href="#" class="icons"><i class="fa-brands fa-github"></i></a>
-                </li>
-                <li>
-                    <a href="#" class="icons"><i class="fa-solid fa-hand-peace"></i></a>
-                </li>
-            </ul>
+        <section class="content">
+            <section class="headline-section">
+                <h1>I'm Massam.<br>Glad to see <br>you!</h1>
+                <p class="text">I’m a software developer! I can help you build a product , feature or website 
+                    Look through some of my work and experience! If you like what you see and have 
+                    a project you need coded, don’t hesitate to contact me.
+                </p>
+                <label class="connect">LET'S CONNECT</label>
+                <ul class="social-media-icons">
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-linkedin"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-sharp fa-solid fa-m"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-github"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-solid fa-hand-peace"></i></a>
+                    </li>
+                </ul>
+            </section>
```

Comment:
- [Optional] COnsider using an indentation of two spaces in all your documents. This is a conventional and also best practice.